### ISSUE TITLE
The Chancellor does entirely the wrong thing!!!

### ISSUE THREAD

**Issue by eshapiro42** [eshapiro42/dominion-game#48]
```
class Chancellor(ActionCard):
    name = 'Chancellor'
    cost = 3
    types = [CardType.ACTION]
    image_path = ''

    description = '\n'.join(
        [
            # '+2 $',
            'You may immediately put your deck into your discard pile.'
        ]
    )

    extra_cards = 0
    extra_actions = 0
    extra_buys = 0
    extra_coppers = 2

    def action(self):
        prompt = f"Would you like to put your deck ({s(len(self.owner.deck), 'card')}) into your discard pile ({s(len(self.owner.discard_pile), 'card')})?"
        if self.interactions.choose_yes_or_no(prompt):
            self.game.broadcast(f'{self.owner} put their deck into their discard pile.')
            self.owner.deck.extend(self.owner.discard_pile)
```

This is so wrong I don't know how I ever could have written it. As is, the action ends up creating duplicates of every card currently in the player's discard and adds those duplicates into the player's deck while leaving the discard pile alone. This is a travesty.

**Comment by eshapiro42** [comment]
A test case like #46 would have caught this instantly.

**Comment by eshapiro42** [comment]
Fixed by ae5373e7e1d47214a0e9829b78d8bbdb494a2d43.
